WordPress/jetpack_metadata/release_notes.txt

```diff
@@ -1,4 +1,4 @@
-- See text blocks in the block editor adjust with changing font size and line height.
+- Font size and line height support for block themes.
```

Comment:
🎉  That makes the notes even shorter now, going from 354 to 322 characters!

Even leaves enough room to make that a bit more sentence-y in the "long" notes like here if you want (while keeping the concise wording for the "short" alternative)?  🤷  Up to you, really, I'm fine with both.
```suggestion
- Font size and line height are now supported for block themes.
```
```suggestion
- Block themes now support font size and line height adjustments
```

Reply:
Oh nice! That would be better yes, thanks for the suggestion. I see the metadata strings were updated already. Am I too late? 😅 I'm fine with either option though.

Reply:
Thanks! Updated!

Reply:
Coolio, I am seeing you update the `long` version of the `Jetpack` release notes only. Is that on purpose @geriux ?

Reply:
My bad 😅 , I've just updated the WordPress one too.

Reply:
😅 Great, it is looking good now, thank you! 🌟